docs/development/extensions-core/druid-lookups.md

```diff
@@ -139,7 +139,7 @@ Off heap cache is backed by [MapDB](http://www.mapdb.org/) implementation. MapDB
 
 |Field|Type|Description|Required|default|
 |-----|----|-----------|--------|-------|
-|maxStoreSize|double|maximal size of store in GB, if store is larger entries will start expiring|no |0|
```

Comment:
There's no neither javadoc nor explicit code in druid, but according to the implementation of underlying library(org.mapdb.DB), it's `GiB`